test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -84,8 +84,11 @@ trait TestData extends ModelGenerators {
     periodKey = periodKey
   )
 
-  val obligationDetails2 = obligationDetails.copy(periodKey = periodKey2)
-  val obligationDetails3 = obligationDetails.copy(periodKey = periodKey3)
+  val obligationDetails2          = obligationDetails.copy(periodKey = periodKey2)
+  val obligationDetails3          = obligationDetails.copy(periodKey = periodKey3)
+  val obligationDetailsFromFuture =
+    obligationDetails.copy(inboundCorrespondenceToDate = LocalDate.now().plusDays(1000))
```

Comment:
@simondrugan16  @yasaracar  - Is there a specific reason this is 1000 days? Do we need this to be configurable?